addons/thtracker/thtracker.lua

```diff
@@ -27,11 +27,12 @@
 
```

Comment:
You should update the copyright notice for the current year, as well as replace the `(c)` with `©`.